.gitignore

```diff
@@ -9,3 +9,4 @@
 /vendor/
 
 .phpunit.result.cache
```

Comment:
does phpunit 10 still create this file name, or can we drop this from ignores?

Reply:
I did run the migrration script. I changed it now to use the default one.
The behaviour of result cache changed some times in PHPUnit 10.